style.css

```diff
@@ -0,0 +1,164 @@
+html {
+  font-size: 10px;
+}
+
+body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;    
+}
+
+.textarea {
+  position: absolute; 
+  top: 30px; 
+  width: 500px;  
+  height: 200px;  
+  resize: none;
+}
+
+.textarea:hover {
+  cursor: no-drop;
+}
+
+.not {
+  user-select: none;
+}
+
+.keyboard {
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  padding: 5px 0;
+  background: #4a545c;
+  box-shadow: 0 0 50px rgba(0, 0, 0, 0.5);
+  user-select: none;
+  transition: bottom 0.4s;
+}
+
+.keyboard__keys {
+  display: flex;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  width: 910px;
+}
+
+.keyboard__key {
+  display: inline-flex;
+  justify-content: center;
+  position: relative;
+  margin: 5px;
+  padding: 0;
+  height: 45px;
+  width: 50px; 
+  border-radius: 4px;
+  border: none;
+  background: rgba(255, 255, 255, 0.2);
+  color: #ffffff;
```

Comment:
can write #fff

Reply:
I try to use the code guide from "HTML Academy". They advise not to reduce color value.
In practice, do these values ​​reduce?

Reply:
It depends on project.
You can write #fff or #ffffff it not necessary.